packages/meditrak-app/app/sync/Synchroniser.js

```diff
@@ -28,7 +31,7 @@ const MEASURE_BATCH_IN_DATA = 'kilobytes';
 const KILOBYTE = 1024; // bytes
 
 const MIN_BATCH_RECORDS = 1;
-const MAX_BATCH_RECORDS = 500;
+const MAX_BATCH_RECORDS = 2000;
```

Comment:
Based on the api_request_log, the majority of our syncs manage to hit this 500 max, so I bumped it to 2000. This also gets hit pretty easily on my local, but the app write speed is balanced with the query time more effeciently 👍